examples/contract-sdk/c10l-hello-world/Cargo.lock

```diff
@@ -0,0 +1,3544 @@
+# This file is automatically @generated by Cargo.
```

Comment:
Hm what was the reason for these examples lock files not being comitted before?

Reply:
Examples never actually had Cargo.lock files, because we wanted to always test it against the latest versions of packages. Will remove those.